lib/improver/cli/spot_extract.py

```diff
@@ -68,6 +68,14 @@ def main(argv=None):
     parser.add_argument("diagnostic_filepath", metavar="DIAGNOSTIC_FILEPATH",
                         help="Path to a NetCDF file containing the diagnostic "
                              "data to be extracted.")
+    parser.add_argument("temperature_lapse_rate_filepath",
```

Comment:
When you load this cube in do you want to check it is a lapse rate cube, or will it fail somewhere else if it isn't.

Reply:
I think this would be better as an option, since this is what it really is - otherwise we end up with quite confusing interfaces.

Reply:
As discussed, because of the use of the taskfilelist generator, this is not possible.